react-frontend/src/components/Home/bcLivesteam.js

```diff
@@ -0,0 +1,129 @@
+import React, { useContext } from 'react';
+import dateFormat from 'dateformat';
+import { Col, Row } from 'react-flexbox-grid';
+import { PageContainer } from '../StyleComponents/pageContent';
+import { Heading } from '../StyleComponents/headings';
+import Query from '../Query';
+import { LinkExternalButton } from '../StyleComponents/htmlTags';
+
+import LIVESTREAM_QUERY from '../../queries/digitalBCLivestream/livestreams';
+import { AppConfigContext } from '../../providers/AppConfig';
+const youTubeLogo = require('../../images/icons/yt_logo_rgb_light.png').default;
+
+const BCLivestream = () => {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <Query query={LIVESTREAM_QUERY}>
+      {({ data: { digitalLivestreamContact, digitalBcLivestreams } }) => {
+        const imageSource =
+          strapiMediaUrl + digitalBcLivestreams[0]?.EventImage?.url;
+        // if no next event is provided in strapi, this section will not render
+        const now = new Date();
+        const nextEventTime = new Date(digitalBcLivestreams[0]?.BroadcastTime);
+        if (nextEventTime && nextEventTime > now) {
+          return (
+            <div style={{ backgroundColor: '#fff' }}>
+              <PageContainer
+                style={{ backgroundColor: '#fff', paddingTop: '40px' }}
+              >
+                <Row between="xs">
+                  <Col sm={12} md={3}>
+                    <Heading>#DigitalBC Livestream</Heading>
+                    <p>
+                      Watch live broadcasts of monthly updates, webinars, sprint
+                      reviews, and more!
+                    </p>
+                    {/* Only render if the link is provided */}
+                    {digitalLivestreamContact?.SubscriptionURL && (
+                      <LinkExternalButton
+                        href={digitalLivestreamContact?.SubscriptionURL}
+                        style={{ maxWidth: '160px' }}
+                      >
+                        Subscribe
+                      </LinkExternalButton>
+                    )}
+                  </Col>
+                  <Col sm={12} md={5}>
+                    <img src={imageSource} style={{ width: '100%' }} />
+                  </Col>
+                  <Col sm={12} md={4} style={{ textAlign: 'center' }}>
```

Comment:
Items in this column need a lot of styling fiddling to match the wireframe:
![Frame 5](https://user-images.githubusercontent.com/16946297/134436040-7e505ca0-b61e-4e04-b3df-b2f8305c721c.png)